src/lib/accessibility/Behaviors/List/SelectableListItemBehavior.ts

```diff
@@ -1,3 +1,4 @@
+import * as keyboardKey from 'keyboard-key'
```

Comment:
thinking about the list use case again - probably list will be used with some kind of virtualized list in the future, which would require support of children API. maybe radiogroup and radioitem would be better use cases for this.

Reply:
yes, that is what we want to support, I haven't yet done the full investigation of how (and if) it will be possible to do. But that is definitely the thing I'll do the next.